src/components/tables/cells/EditTask.tsx

```diff
@@ -0,0 +1,22 @@
+import { Button } from '@mui/material';
+import { FormattedMessage } from 'react-intl';
+
+interface Props {
+    clickHandler: () => void;
+}
+
+function EditTask({ clickHandler }: Props) {
```

Comment:
This component was previously a table cell, but is now an item of a menu presented only in a table cell.

Since the `OptionsMenu` contains a `TableCell` wrapper, creating a subdirectory within _src/components/tables/cells_ to house it and its menu items could make sense. Consequently, the internal `TableCell` wrapper could be removed from `OptionsMenu` and the menu component and its children could be moved to a unique subdirectory of _src/components/menus_.